src/openrct2-android/app/src/main/CMakeLists.txt

```diff
@@ -0,0 +1,131 @@
+
+#
+# Copyright (C)  The Android Open Source Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+cmake_minimum_required(VERSION 3.4.1)
+
+set(CMAKE_VERBOSE_MAKEFILE on)
+
+# build native_app_glue as a static lib
+# include_directories(${ANDROID_NDK}/sources/android/native_app_glue)
+# add_library(app-glue STATIC
+#            ${ANDROID_NDK}/sources/android/native_app_glue/android_native_app_glue.c)
+
+
+set(lib_src_DIR ${CMAKE_SOURCE_DIR}/../../../libs)
+set(lib_build_DIR $ENV{HOME}/tmp)
+file(MAKE_DIRECTORY ${lib_build_DIR})
+
+set(DEBUG_LEVEL 0 CACHE STRING "Select debug level for compilation. Use value in range 0–3.")
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -DDEBUG=${DEBUG_LEVEL}")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DDEBUG=${DEBUG_LEVEL}")
+
+
+include(ExternalProject)
+ExternalProject_Add(libs
+    URL https://github.com/marijnvdwerf/openrct2-dependencies-android/releases/download/v0.3/openrct2-libs-android-${ANDROID_ABI}.zip
```

Comment:
All these libraries should be published as a release in https://github.com/OpenRCT2/Dependencies/releases

It would be nice if we had some document detailing how to build them, too